src/main/locales/localisation.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#if defined(USE_OSD_HD)                         // HD screen
+  #define TR2(x, y) y
+#elif defined(USE_OSD) || defined(USE_OSD_SD)   // PAL or NTCS screen
+  #define TR2(x, y) x
+#else
+  #define TR2(x, y) x
+#endif
```

Comment:
I don't think this works. By default, we add the `USE_OSD_HD` to a lot of FC by default. So the user will see the HD version of strings always, it does not depend if the user has it configured as SD or HD.
I don't see a way to fix this in build time, except make it mandatory to select between SD or HD to the user while flashing, but I don't think that this a good idea.

Reply:
Hi @McGiverGim , right, I will try to make a solution, but have som exams the next 2 weeks, but then :-)
A runtime shift, seems to be at better solution, I believe this will end good ...

Reply:
hi @McGiverGim , just a comment to this, now USE_HD_EXTENDED are the compiler flag